### PR TITLE
Add porthole@1.0.0 w/ git auto-update

### LINKED ISSUE #11096: [Request] Add porthole
**Library name:** porthole
**Git repository url:** https://github.com/ternarylabs/porthole
**npm package name or url** (if there is one): 
**License (List them all if it's multiple):** MIT
**Official homepage:** http://ternarylabs.github.com/porthole
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by dakshshah96**
Pull request for issue: #11096 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ternarylabs/porthole
 * Official website (optional, not the repository): http://ternarylabs.github.com/porthole
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 42
   - Count of stars: 639
   - Count of forks: 101

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@dakshshah96 Could you ask the author can add a tag for it?
Because the CDNJS add auto-update by version tags of the library. And there is no git tags on GitHub repository of this library.
Here is an example --> https://github.com/moappi/json2html/issues/22

Thank you!

**Comment by dakshshah96** [comment]
@sufuf3 Communicated git tags requirement to author in [#48 on porthole](https://github.com/ternarylabs/porthole/issues/48).

**Comment by dakshshah96** [comment]
@sufuf3 @extend1994 @hare1039 The author has added the v1.0.0 tag. 😄
Please remove the `Wait for response` label from this PR.

**Comment by dakshshah96** [comment]
@qwe19272375 Please add `in progress` label here.

File: ajax/libs/porthole/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "porthole",
+  "version": "1.0.0",
+  "filename": "porthole.min.js",
+  "description": "A proxy to safely communicate to cross-domain iframes in JavaScript.",
+  "homepage": "https://ternarylabs.github.com/porthole/",
```
Comment (pvnr0082t) [inline ajax/libs/porthole/package.json]: https://ternarylabs.github.com/porthole/ will redirect to http://ternarylabs.github.io/porthole/. I think let's use https://ternarylabs.github.io/porthole/ here.

**Comment by dakshshah96** [comment]
@pvnr0082t Done 👍

**Comment by PeterDaveHello** [comment]
I think you need to cc its author, not the organization, otherwise 👍

**Pull request merged**